eva_sub_cli/validators/validator.py

```diff
@@ -445,6 +457,52 @@ def _convert_metadata_attribute(self, sheet, json_attribute, xls2json_conf):
             if attributes_dict[attribute] == json_attribute:
                 return attribute
 
+    def _collect_md5sum_to_metadata(self):
+        md5sum_file = resolve_single_file_path(os.path.join(self.output_dir, 'md5sums.txt'))
+        file_path_2_md5 = {}
+        file_name_2_md5 = {}
+        if md5sum_file:
+            with open(md5sum_file) as open_file:
+                for line in open_file:
+                    sp_line = line.split()
+                    md5sum = sp_line[0]
+                    vcf_file = sp_line[1]  # assuming no space in the vcf_file_name
+                    file_path_2_md5[vcf_file] = md5sum
+                    file_name_2_md5[os.path.basename(vcf_file)] = md5sum
```

Comment:
I assume this is to accommodate validating in Docker where full paths within the container don't line up with those outside the container... I'm not sure the basename is enough, for example if the submitter has two files:
* `~/project/analysis_1/file.vcf`
* `~/project/analysis_2/file.vcf`

and is running the CLI from within `~/project`.

I feel like we either need to strip the prefix that Docker adds, or maybe use a relative path, to be robust to this kind of thing...

Reply:
Actually I just tried this (2 vcfs with the same basename but different full paths) and it validates both files but only reports on one, so if we want to support this there are probably a few things that need to change.

Reply:
The two different dicts is to support both file name and file path in the metadata json
The docker path is supported using `self._validation_file_path_for(vcf_file)`
I'll try out with two file with the same basename.

Reply:
Okay I get it, I think the same basename issue is unrelated to this PR (but might need a new ticket...)